Makefile

```diff
@@ -82,6 +82,12 @@ verify-scripts:
 .PHONY: verify-scripts
 verify: verify-scripts verify-codegen-crds
 
+deploy-operator: ensure-operator-sdk
```

Comment:
could we change this task name? e.g. deploy-with-olm or else, this name may not be straightforward

Reply:
done